flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/InfluxdbReporterOptions.java

```diff
@@ -57,6 +59,13 @@
 		.defaultValue("")
 		.withDescription("(optional) the InfluxDB retention policy for metrics");
 
+	public static final ConfigOption<InfluxDB.ConsistencyLevel>  CONSISTENCY = ConfigOptions
+		.key("consistency")
+		.enumType(InfluxDB.ConsistencyLevel.class)
+		.defaultValue(InfluxDB.ConsistencyLevel.ONE)
+		.withDescription(String.format("(optional) InfluxDB consistency level for metrics. default is %s",
```

Comment:
the default is also automatically documented; so this can just say "(optional) the InfluxDB consistency level for metrics